packages/vite/src/node/plugins/css.ts

```diff
@@ -2046,7 +2046,7 @@ function fixScssBugImportValue(
   return data
 }
 
-// .scss/.sass processor
+// #region .scss/.sass processor
```

Comment:
I think we could have `Sass`, `Less`, `Stylus` as the region names if we merge this. It is better to be consistent (Sass already implies both .scss and .sass, and we also don't need the word processor in the first one)

Reply:
Okay.